README.md

```diff
@@ -8,157 +8,147 @@
 [![Coverage Status](https://coveralls.io/repos/github/Unleash/unleash-edge/badge.svg?branch=main)](https://coveralls.io/github/Unleash/unleash-edge?branch=main)
 ![downloads](https://img.shields.io/crates/d/unleash-edge.svg)
 
-> Warning: Unleash Edge requires Unleash v4.15.0 or higher
+- [Overview](#overview)
+- [Quickstart](#quickstart)
+- [Edge behaviors](#edge-behaviors)
+  - [Strict behavior](#strict-behavior)
+  - [Dynamic behavior](#dynamic-behavior)
+- [Getting Unleash Edge](#getting-unleash-edge)
+- [Running Unleash Edge](#running-unleash-edge)
+- [Metrics](#metrics)
+  - [Prometheus integration](#prometheus-integration)
+- [Compatibility](#compatibility)
+- [Debugging](#debugging)
+- [Additional resources](#additional-resources)
+  - [Edge concepts](#edge-concepts)
+  - [CLI](#cli)
+  - [Deploying Edge](#deploying-edge)
+  - [Migrating from Unleash Proxy](#migrating-from-unleash-proxy)
+  - [Performance benchmarking](#performance-benchmarking)
+  - [Contribution and development guide](#contribution-and-development-guide)
 
-Unleash Edge is the successor to the [Unleash Proxy](https://docs.getunleash.io/how-to/how-to-run-the-unleash-proxy).
```

Comment:
Didn't feel like the most important thing to start with. Thought we could explain what the product is first, and mention the fact that it's a successor later on in this section.